docs/en/sql-reference/functions/other-functions.md

```diff
@@ -300,11 +300,97 @@ The argument is internally still evaluated. Useful e.g. for benchmarks.
 
 ## sleep(seconds)
 
-Sleeps ‘seconds’ seconds for each data block. The sleep time can be specified as integer or as floating-point number.
+Used to introduce a delay or pause in the execution of a query. It is primarily used for testing and debugging purposes.
+
+**Syntax**
+
+```sql
+sleep(seconds)
+```
+
+**Arguments**
+
+- `seconds`: [Int](../../sql-reference/data-types/int-uint.md) The number of seconds to pause the query execution to a maximum of 3 seconds. It can be a floating-point value to specify fractional seconds.
```

Comment:
Why int if you say that it can be floating point in the same paragraph?

Reply:
Fix: https://github.com/ClickHouse/ClickHouse/pull/61308